libmodulemd.spec.in

```diff
@@ -1,13 +1,5 @@
 %global libmodulemd_version @VERSION@
-%global libmodulemd_v1_version @V1_VERSION@
-
-%if ( 0%{?fedora} && 0%{?fedora} <= 30 ) || ( 0%{?rhel} && 0%{?rhel} <= 7)
-%global build_python2 1
```

Comment:
The usage of `%if %{build_python2}`  below now needs to be dropped.

Reply:
That was a mistake. I only should have deleted the meson flags, not  `%build_python2`. I just pushed a fix.